.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+[*]
+insert_final_newline = true
+
+[*.{kt,kts}]
+max_line_length = 140
+indent_size = 4
+ij_kotlin_allow_trailing_comma = true
+ij_kotlin_allow_trailing_comma_on_call_site = true
+
+[*.java]
+ij_java_class_count_to_use_import_on_demand = 3
+ij_java_names_count_to_use_import_on_demand = 3
```

Comment:
Is this the threshold to use * imports? If so, let's configure this to be 9999 or off somehow.

Reply:
I'm not the biggest fan of star imports as well 😄 I now set it to the maximum value, both for Java and for Kotlin files. 

And yes, this was the threshold, I remembered the default value was 3, plus checked the committed [codestyle](https://github.com/airbnb/lottie-android/blob/master/.idea/codeStyles/Project.xml) file doesn't contain any other overrides.